build/ci/github_actions/operator/save_driver_pods_logs.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+
+get_all_pods_by_type (){
+    pod_type=$1
+    kubectl get pod -l csi | grep $pod_type | awk '{print$1}'
+}
+
+run_action_on_pod (){
+    pod_type=$1
+    action=$2
+    extra_args=$3
+    get_all_pods_by_type=$(get_all_pods_by_type $pod_type)
+    kubectl $action $get_all_pods_by_type $extra_args > "/tmp/driver_${get_all_pods_by_type}_${action}.txt"
```

Comment:
where is this available? could you write an example link in this conversation?
would the name be something like `driver_ibm-block-csi-node-gcbw2ibm-block-csi-node-lv29m_get.txt`?

Reply:
thanks!
1. perhaps the name `save_driver_logs_to_artifacts.sh` would convey this info.
2. looking at the zip, I see some gaps
![image](https://user-images.githubusercontent.com/57443811/127774517-b3465257-fb81-477a-bed0-b37aff80094d.png)
a. no need for `driver_`
b. sidecar logs are missing
c. `describe pod` has a space, in contrast to the rest of the word separators, which is unconventional, maybe use an `action_name` which is the first word of the `action`?

Reply:
2.b. yes, we want to make sure the operator provided all their required permissions, flags, etc...

Reply:
I still don't see any sidecar logs.
and the zip name still contains a space (and IMO should not be capitalized at all)

Reply:
done

Reply:
now the `driver_logs` is with an underscore (which I think is good), but `ci-dependencies` filename is still with a hyphen.
and IMO no need for `_container` in the logs filename.

Reply:
> now the `driver_logs` is with an underscore (which I think is good), but `ci-dependencies` filename is still with a hyphen.
 done

Reply:
> now the `driver_logs` is with an underscore (which I think is good), but `ci-dependencies` filename is still with a hyphen.
>
 done

> and IMO no need for `_container` in the logs filename.
>
and the structure will be: `{pod_name}_{container_name}_{action_name}`? 
the `_container` doesn't add more information to the log file?

> and since each file in the zip is a single log, I would rename `_logs.txt` to `log.txt` or even `.log`
>
I think I will change to `log.txt`

Reply:
> and the structure will be: `{pod_name}_{container_name}_{action_name}`?
> the `_container` doesn't add more information to the log file?

I'm not sure `describe` is relevant to a specific container.
and IMO the sidecar names are indicative (and long) enough, so `_container` doesn't add more information (same reason we don't use `_pod` before the pod name)


> I think I will change to `log.txt`
great